tests/test_create_command_docs.py

```diff
@@ -0,0 +1,76 @@
+import os
+from unittest import TestCase
+
+from click.testing import CliRunner
+
+from tests.conftest import BASE_DIR
+from utils.create_command_docs import docs
+
+
+class TestCreateCommandDocsCli(TestCase):
+    def setUp(self) -> None:
+        self.runner = CliRunner()
+
+    @classmethod
+    def tearDownClass(cls):
+        os.remove(f"{BASE_DIR}/tests/fixtures/docs.md")
+
+    def test_check_required_module_option(self):
+        result = self.runner.invoke(docs)
```

Comment:
Might be worth passing the `cmd` parameter here to be sure it's failing for the right reason. E.g. pass everything except the one we are testing.

Comment applies on the others probably too.

Reply:
Definitely worth doing that. Resolved in https://github.com/uktrade/copilot-tools/pull/83/commits/7ac92954f136625fd97c0bb01fe97877a3539733.